krmllib/Makefile

```diff
@@ -6,6 +6,12 @@
 
 all: verify-all compile-all
 
+extra:
+	+dune build
```

Comment:
1. based on my understanding of the GNU Make manual, section "9.3 Instead of Executing Recipes": adding a `+` here means that if someone runs, e.g., `make -n extra` to see what *would* be done by make, then `dune` is *still* invoked... do I understand this right, and is this a desired behavior?

2. do you know who uses C.ml and TestLib.ml? I don't, and the last reference to them comes from 2017. Can we just remove those files, and the corresponding targets and dune-project files?